src/_includes/head.html

```diff
@@ -115,21 +124,6 @@
 <link rel="canonical" href="{{ page.canonical.href }}" />
 {% endif %}
 
-<script>
-    window.codurance = window.codurance || {};
```

Comment:
Are we getting rid of this object entirely? If so, should we delete `onLunrLoaded` from the 404.html as well?